editor/components/block-settings-menu/test/reusable-block-settings.js

```diff
@@ -19,7 +19,7 @@ describe( 'ReusableBlockSettings', () => {
 		);
 
 		const text = wrapper.find( 'IconButton' ).children().text();
-		expect( text ).toEqual( 'Convert to Reusable Block' );
+		expect( text ).toEqual( 'Convert to Shared Block' );
```

Comment:
What about file names? Should we update them, too?

Reply:
Internally we're still referring to these things as _reusable blocks_. My feeling is that the name has traction within the team, e.g. in GitHub we have a _Reusable Blocks_ label. Happy to go on a renaming spree if we think it's worth doing, though.

Reply:
Yeah, we should rename everything, as well. We won't be able to do it after merge, so better to do it before.

Reply:
Yes, let's rename.